### ISSUE TITLE
The detail of Bart program model

### ISSUE THREAD

**Issue by Snnzhao** [shijx12/KQAPro_Baselines#5]
In the train.py, the model is not valid during training. So I copy the valid part from predict.py.
But  during training at start, the valid acc drop from 0.007 to 0.004 and does not increase during training.
I use bart-base as pretrained model.
Can you provide more details for the experiment?

**Comment by Snnzhao** [comment]
I found even if train 25 epochs when the loss drop to 0.003.
It outputs a lot program like  
"Find(farmer)\<b>Relate(occupation\<c>backward)\<bb>Find(George Segal)\<B>And()\<b>>Find(Rutgers University)\< b>QueryRelation()"
While the target is 
"Find(farmer)\<b>Relate(occupation<c>backward)\<b>Find(George Segal)\<b>And()\<b>Find(Rutgers University)\<b>QueryRelation()"

**Comment by Snnzhao** [comment]
It sames the split token always wrong

**Comment by Snnzhao** [comment]
I sames to solve this problem.
This problem is caused by the token \<b> \<c>, whick is not included in pretrained vocab.
I solve this problem by replace this two with # and |.
Thanks again for such wonderful work.

**Comment by shijx12** [comment]
It is due to the error in the latest BART checkpoint. If somebody meets this problem, you can download the previous checkpoint from <https://cloud.tsinghua.edu.cn/f/3b59ec6c43034cfc8841/?dl=1> and load it as the pretrained model to achieve our reported results.
